File: app/src/main/java/com/openchat/openchat/Message.java
package com.openchat.openchat;

/**
 * Created by Dan on 14/03/2018.
 */

public class Message {

    private String content;
    private String name;
    private String time;

    public Message() {

    }
    public Message(String content) {
        this.content = content;

    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }
}

File: app/src/main/java/com/openchat/openchat/Message.kt
package com.openchat.openchat

/**
 * Created by Dan on 14/03/2018.
 */

class Message {

    var content: String? = null
    var name: String? = null
    var time: String? = null

    constructor() {

    }

    constructor(content: String) {
        this.content = content

    }
}